lib/tasks/copyable.rake

```diff
@@ -39,7 +39,8 @@ task :copyable => :environment do
 
   all_associations = model_class.reflect_on_all_associations
   required_associations = all_associations.select do |ass|
-    (ass.macro != :belongs_to) && ass.options[:through].blank?
+    !ass.is_a?(ActiveRecord::Reflection::BelongsToReflection) &&
+    !ass.is_a?(ActiveRecord::Reflection::ThroughReflection)    
```

Comment:
Interesting that this block is so duplicated between this rake task and the association checker.

Reply:
yah i have no idea why